### DISCUSSION TITLE
CopyConfig.cmake.in suggestion

### DISCUSSION THREAD

**Discussion by therion23** [jcorporation/myMPD#1079] [Ideas]
Hello!

I build and run myMPD as my own user, installed to /usr/local as the advanced build describes, except for the sudo part. That means "make install" insists on writing a system wide service file, which it obviously cannot.

Would it be possible to modify this, so CopyConfig either tests for write permissions first, or makes the failed file copy a non fatal? Or maybe even a cmake option to skip installing startup scripts entirely?

It is simply to keep my own builds isolated from the system packages.

(Not really a bug, so i put it here - hope that's correct).

Thank you in advance, and have a great day!
Bo

**Comment by jcorporation** [comment]
I added a cmake option to disable the installation of a startup script.

**Reply by therion23** [reply]
You are such a gent - thank you!

**Reply by therion23** [reply]
May i ask what it is called? Cannot seem to find it.

**Reply by jcorporation** [reply]
It is MYMPD_STARTUP_SCRIPT. But it is currently only in the v12.0.0 branch.
